manifests/init.pp

```diff
@@ -226,7 +226,7 @@
   String[1] $db_password = $foreman::params::db_password,
   Optional[String[1]] $db_sslmode = undef,
   Optional[String[1]] $db_root_cert = undef,
-  Integer[0] $db_pool = 5,
+  Integer[0] $db_pool = 9,
```

Comment:
Should we make this:
```suggestion
  Optional[Integer[0]] $db_pool = undef,
```

That way, if we change it in the future it automatically gets recalculated. We would need to change the part in `config.pp` to deal with it by using `pick()` instead of `max()`.

The DB migration could then simply be `.delete('db_pool') if db_pool == 5`.